File: CS50x/Pset6/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    spaces = height - i - 1
    hashes = i + 1

    print(" " * spaces, end="")
    print("#" * hashes, end="")
    print()

File: CS50x/Pset1/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void) {
    int n;
    n = get_int("Height :");

    while (n <= 0 || n > 8) {
        n = get_int("Height :");
    }

    for (int i = 1; i <= n; i++) {
        for (int j = 0; j < n - i; j++) {
            printf(" ");
        }
        for (int k = 0; k < i; k++) {
            printf("#");
        }
        printf("  ");
        for (int l = 0; l < i; l++) {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}
